test/integration/alerting/cases.yaml

```diff
@@ -0,0 +1,9 @@
+- id: alert-01
```

Comment:
I must say, I liked the suffixes of previous cases, from which one could somewhat glean what they tested. If I know alert-01 failed, it tells me nothing, with alert-01-cs I might know something. This isn't really interesting for the usual testing, but if many things fail and you look into the k8s logs, you'll see things like `Pod alert-01` was deployed. Ok cool, I have 13 failing cases, was this one of them and which one is it?